contrib/endpoints/src/api_manager/check_security_rules_test.cc

```diff
@@ -0,0 +1,413 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+//
+
+#include "contrib/endpoints/src/api_manager/check_security_rules.h"
+#include "contrib/endpoints/src/api_manager/context/request_context.h"
+#include "contrib/endpoints/src/api_manager/context/service_context.h"
+#include "contrib/endpoints/src/api_manager/mock_api_manager_environment.h"
+#include "contrib/endpoints/src/api_manager/mock_request.h"
+
+using ::testing::_;
+using ::testing::AllOf;
+using ::testing::Invoke;
+using ::testing::Property;
+using ::testing::Return;
+using ::testing::StrCaseEq;
+using ::testing::StrEq;
+using ::testing::StrNe;
+
+using ::google::api_manager::utils::Status;
+using ::google::protobuf::util::error::Code;
+
+namespace google {
+namespace api_manager {
+
+namespace {
+const char kServiceName[] = R"(
+name: "myfirebaseapp.appspot.com")";
+
+const char kBadServiceName[] = R"(
+name: "badService.appspot.com")";
+
+const char kProducerProjectId[] = R"(
+producer_project_id: "myfirebaseapp"
+)";
```

Comment:
Can we make it consist with the two cases above by merging this line to the end of previous line?